### PR TITLE
Achievement/cobra

### PR DESCRIPTION
**Pull request by d12frosted**
Simply as that. 

Following #5476 

---

Also he might be Wiki Master, or something like that. Like a part job :D

### CODE REVIEW

**Comment by cpaulik** [comment]
Are you trying to sneak some keybindings in? :wink:

**Comment by d12frosted** [comment]
Oops. My bad :) wrong branch to move from ;)

Trimis de pe iPad-ul meu

Pe 14 mar. 2016, la 22:22, Christoph Paulik <notifications@github.com> a scris:

**Comment by d12frosted** [comment]
Sneaking? Sneaking?! d12frosted means no sneaking!

**Comment by syl20bnr** [comment]
Waiting for others to propose other tittles.

File: doc/DOCUMENTATION.org [hunk 1]
```diff
@@ -2818,6 +2818,7 @@ with ~SPC SPC spacemacs/recompile-elpa~.
 | The Expert in Latin Language ([[https://github.com/syl20bnr/spacemacs/pull/4043][PR]])               | [[https://github.com/vijaykiran][vijaykiran]]   |
 | The Tiler ([[https://github.com/syl20bnr/spacemacs/pull/4068][eyebrowse integration]])               | [[https://github.com/bmag][bmag]]         |
 | The Geometer ([[https://github.com/TheBB/spaceline][spaceline]])                        | [[https://github.com/TheBB][TheBB]]        |
+| The Lord of the Issues                          | [[https://github.com/StreakyCobra][StreakyCobra]] |
```
Comment (robbyoconnor) [inline doc/DOCUMENTATION.org]: :+1:

Comment (robbyoconnor) [inline doc/DOCUMENTATION.org]: Lord of fhe Issues makes him sound like a guy with a lot of er...issues.

Comment (nixmaniack) [inline doc/DOCUMENTATION.org]: I was thinking the same. :smile:

**Comment by TheBB** [comment]
The Ticketmaster

**Comment by robbyoconnor** [comment]
Ticketmaster will sue us...

**Comment by beta1440** [comment]
What exactly is the achievement for?

**Comment by robbyoconnor** [comment]
@beta1440 @StreakyCobra  is amazing and important to spacemacs. We tend to award achievements to people who have contributed to the project.

**Comment by sdwolfz** [comment]
I resurrect this PR and vote for `Ticketmaster` as well.

(will take over and open a new PR but merge it as a contribution from @d12frosted)

**Comment by robbyoconnor** [comment]
@delaanthonio Not sure if anyone addressed you -- suspect you changed your username but those who do amazing contributions tend to get recognized for their work..do amazing work and you, or anyone could wind up with one :)

**Comment by WarFox** [comment]
Let's get it done 👍

**Comment by delaanthonio** [comment]
@robbyoconnor yes, I did change my username. I used to be @beta1440. Nevertheless, thanks for the response.

**Comment by robbyoconnor** [comment]
It is a beautiful way to encourage people to contribute and also recognize those who do good work :)

**Comment by d12frosted** [comment]
@sdwolfz I've updated my PR ❤️

**Comment by sdwolfz** [comment]
Thank you :heart:!
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by StreakyCobra** [comment]
So much :heart: Thank you all for this :blush:

**Pull request merged**